test/hotspot/jtreg/compiler/lib/ir_framework/CompilePhase.java

```diff
@@ -60,6 +60,26 @@ public enum CompilePhase {
     ITER_GVN_AFTER_VECTOR("Iter GVN after vector box elimination"),
     BEFORE_BEAUTIFY_LOOPS("Before beautify loops"),
     AFTER_BEAUTIFY_LOOPS("After beautify loops"),
+    BEFORE_LOOP_UNROLLING("Before Loop Unrolling"),
+    AFTER_LOOP_UNROLLING("After Loop Unrolling"),
+    BEFORE_SPLIT_IF("Before Split If"),
+    AFTER_SPLIT_IF("After Split If"),
+    BEFORE_LOOP_PREDICATION_IC("Before Loop Predication IC"),
+    BEFORE_LOOP_PREDICATION_RC("Before Loop Predication RC"),
+    AFTER_LOOP_PREDICATION("After Loop Predication"),
```

Comment:
Would it be possible to define `AFTER_LOOP_PREDICATION_IC` and `AFTER_LOOP_PREDICATION_RC` separately, to match the corresponding `BEFORE_*` phases? This would make it easier to understand what is the scope of each transformation in the IGV outline and also play better with the enumeration of repeated IGV graph dumps introduced in this changeset. To be more specific, what I would expect to see here:
![igv-pred](https://github.com/openjdk/jdk/assets/8792647/02a11ce7-d41a-43f1-9711-a7a403c678e9)
is:
```
15. Before Loop Predication IC: 90 If
16. After Loop Predication IC: 163 If
17. Before Loop Predication RC: 106 RangeCheck
18. After Loop Predication RC: 196 RangeCheck
```

Reply:
Sounds good to me, this is how we implemented it originally. I see @chhagedorn reacted with a thumbs up, so I'll go ahead with the change!